.github/workflows/require_be_approval.yml

```diff
@@ -7,10 +7,21 @@ on:
 jobs:
   require-backend-approval:
     name: Require Backend Approval
-    if: contains(github.event.pull_request.labels.*.name, 'require-backend-approval')
     permissions: write-all
     runs-on: ubuntu-latest
     steps:
+      - name: Check pr
+        run: |
+          echo "pr_draft=${{ github.event.pull_request.draft }}"
+
+      - name: Draft Pull Request Check
+        if: github.event.pull_request.draft == true
+        run: exit 1
+
+      - name: Check for Ready for Review label
+        if: !contains(github.event.pull_request.labels.*.name, 'require-backend-approval')
+        run: exit 0
```

Comment:
require-backend-approval label is attached when the pull request is ready for review